titus_isolate/allocate/allocate_response.py

```diff
@@ -9,6 +10,10 @@ def __init__(self, cpu: Cpu, cpu_allocator_name: str, metadata: dict = {}):
         self.__cpu = cpu
         self.__metadata = metadata
         self.__metadata[CPU_ALLOCATOR] = cpu_allocator_name
+        if ALLOCATOR_SERVICE_TASK_ID not in self.__metadata:
+            alloc_service_id = os.environ.get(TITUS_TASK_ID, None)
```

Comment:
I would use a marker value like `UNKNOWN_ALLOC_SERVICE_ID` instead of not setting the value.  It's easier to track down who's not reporting the way you expect.

Reply:
thanks, done